public/testem/qunit_adapter.js

```diff
@@ -5,8 +5,8 @@ qunit_adapter.js
 
 Testem's QUnit adapter. Works by using QUnit's hooks:
 
-* `testStart`
-* `testDone`
+* `begin`
+* `done`
```

Comment:
That might be a problem. IIRC Testem needs to hook into each test to report test results incrementally.